app/src/main/res/values/strings.xml

```diff
@@ -248,10 +248,10 @@
     <br/><br/>
     Force closing channels will cost you money (to cover the on-chain fees) and will cause your funds to be locked for days. <b>Do not uninstall the app until your channels are fully closed or you will lose money.</b>
     <br/><br/>
-    Funds will eventually be sent to: %1$s
-    <br/><br/>
     <b>Do not use this feature if you don\'t fully understand what it does.</b>
   ]]></string>
+  <string name="closechannels_force_destination">Funds will eventually be sent to:</string>
+  <string name="closechannels_force_destination_desc">This address is derived from your seed and belongs to you.</string>
```

Comment:
Translation to Portuguese Brazilian: Este endereço foi originado a partir da sua seed e pertence a você.

Obs. Don't you think instead of (is derived) should be (was derived)?

Reply:
Hi @dpad85, any opinion here?

Reply:
Hi @bitcoinuser I will add your changes in a separate commit.

Reply:
Btw I think `is derived` works better than `was derived` here.

Resolved with 1c12311